packages/db/src/pouch/databases.ts

```diff
@@ -176,7 +195,7 @@ export abstract class Databases<C extends Collections> implements Workspace<C> {
         // check for existing
         const resource = await this.get(collectionName, id);
         if (resource) {
-          return resource;
+          return this.merge(collectionName, resource, resourceInput);
```

Comment:
Just realized that you're not saving this - you're just returning it. Probably want to save the merged version!